src/Admin/ReenableCssTransientCachingAjaxAction.php

```diff
@@ -84,7 +85,7 @@ public function register_ajax_script( $hook_suffix ) {
 						} );
 				} );
 			} );
-	} );
+	}, 1000 );
```

Comment:
Why 1 second specifically?

Reply:
It's an estimation to give a bit of leeway for all direct site health tests to be added to the page. One alternative could be to add a `setTimeout` to keep checking for when this specific site health test gets added to the page, but that seems like a waste of CPU cycles.

Reply:
But the test is not async so why would it not be on the page initially?